config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```diff
@@ -21,6 +21,25 @@ postsubmits:
       - name: github
         secret:
           secretName: oauth-token
+  kubernetes/test-infra:
+  - name: post-test-infra-push-prow
+    cluster: test-infra-trusted
+    decorate: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/gcloud-bazel:v20181002-996397d
+        command:
+        - prow/bump.sh
+        env:
+        - name: GOOGLE_APPLICATION_CREDENTIALS
+          value: /creds/service-account.json
+        volumeMounts:
+        - name: creds
+          mountPath: /creds
+      volumes:
+      - name: creds
+        secret:
+          secretName: pusher-service-account
```

Comment:
If/when we add more publishing jobs we should move the configuration for these credentials into a preset similar to `preset-service-account`.